### ISSUE TITLE
@coversNothing triggers risky test

### ISSUE THREAD

**Issue by saschahepp** [sebastianbergmann/phpunit#2163]
With PHPUnit version 5.3.1 when you got a test-case like this:

```
/**
 * @coversNothing
 */
class someTest extends PHPUnit_Framework_TestCase
{
    public function testSomething()
    {
        $this->assertTrue(true);
    }
}
```

and you call phpunit with a config file that sets the option
`checkForUnintentionallyCoveredCode="true"`
you get the output
```
PHPUnit 5.3.1 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.0.6 with Xdebug 2.4.0
Configuration: /var/www/mydevsite/phpunit.xml
Warning:       Deprecated configuration setting "checkForUnintentionallyCoveredCode" used

R                                                                   1 / 1 (100%)

Time: 459 ms, Memory: 18.00Mb

There was 1 risky test:

1) someTest::testSomething
This test does not have a @covers annotation but is expected to have one

OK, but incomplete, skipped, or risky tests!
Tests: 1, Assertions: 1, Risky: 1.
```

When calling

`phpunit --no-configuration --strict-coverage someTest.php`

everything is ok.

It seems like the checkForUnintentionallyCoveredCode-setting does not respect the @coversNothing annotation.
I see the deprecation-warning, but couldn't find any alternative in the [documentation](https://phpunit.de/manual/current/en/phpunit-book.html#risky-tests.unintentionally-covered-code) ;-)

Greetings
Sascha

**Comment by dsiebel** [comment]
+1 just stumbled upon this one, too. Documentation of PHPUnit 5.5 mentions this setting.

**Comment by saschahepp** [comment]
Seems it is somehow fixed for current stable 5.5. Could not reproduce it with this version.

The deprecation warning is still there but the @coversNothing annotation seems to work now.
